pkg/logql/parser_test.go

```diff
@@ -386,6 +386,125 @@ func TestParse(t *testing.T) {
 				nil,
 				newString("5")),
 		},
+		{
```

Comment:
These test cases are just a copy of the above with the duration shifted near the selector.